README.md

```diff
@@ -29,50 +42,51 @@ export async function greeting(/** @type {Context} */ context) {
 }
 ```
 
-To run: `./boltzmann.js`. And to view the response: `http GET localhost:5000/hello/world`
+To run: `./boltzmann.js`. And to view the response: `[http](https://httpie.org) GET localhost:5000/hello/world`
 
 ## Creating a boltzmann app
 
-This repo provides a command-line tool, `ludwig`, that generates a single javascript file with all of Boltzmann's implementation, and some scaffolding for a full Boltzmann app if it's not already present. The tool enables or disables specific Boltzmann features. The tool is safe to re-run, *provided* you are running it in a versioned git directory.
+This repo includes a command-line tool, `ludwig`, that generates a single javascript file with all of Boltzmann's implementation, and some scaffolding for a full Boltzmann app if it's not already present. The tool enables or disables specific Boltzmann features. The tool is safe to re-run, *provided* you are running it in a versioned git directory.
```

Comment:
Rough suggestion here, we might orient this more towards "you want to try this tool. here's how. followed by what to expect."

```
To get started with Boltzmann, download the `boltzmann` command-line tool.
You can get it from [the releases page][] or run it via `npx boltzmann-cli`. The
tool is responsible for initializing a new Boltzmann project as well as keeping it
up to date. You enable or disable specific Boltzmann features using the tool.

To scaffold with redis and honeycomb:
```

Reply:
(tacit in this: while I like the `ludwig` name, I think we might just align the cli tool name to reduce confusion. Also, we should make all the builds available via NPM to make it easier for folks to install – i.e., we shouldn't require users to know their specific architecture + platform.)